service/worker/scanner/build_ids/scavenger.go

```diff
@@ -211,6 +212,11 @@ func (a *Activities) processNamespaceEntry(
 			}
 			entry := tqResponse.Entries[heartbeat.TaskQueueIdx]
 			if err := a.processUserDataEntry(ctx, rateLimiter, *heartbeat, ns, entry); err != nil {
+				if common.IsContextDeadlineExceededErr(err) {
```

Comment:
So you wanna remove the check on line 210?

Reply:
No, I commented on that: https://github.com/temporalio/temporal/pull/4841#discussion_r1317723758
Not sure why github isn't showing it on the diff view

Reply:
I see, I would check `ctx.Err() != nil` here instead, `processUserDataEntry` is what we want to ignore errors for.